functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -424,23 +424,23 @@ def add_training_session(self, training: TrainingSession, user_id: int):
                 cursor = conn.cursor()
                 query = ("WITH t AS ( "
                         "INSERT INTO Training_Info (TrainingDescription, ModelLocation, ClassPerfAvg, CreatedByUser) "
-                        "VALUES ({},{},{},{}) RETURNING TrainingId), "
+                        "VALUES ('{}','{}',{},{}) RETURNING TrainingId), "
```

Comment:
@samiyaakhtar thanks for adding these. I didn't get a chance to make the integration test.